src/aura/BGE_EntryForm/BGE_EntryForm.cmp

```diff
@@ -32,23 +32,28 @@
   @date 2018
   @group BGE
   @group-content ../../ApexDocContent/BatchDataImport.htm
-  @description Component used to view, edit, and delete DataImport__c records on the related NPSP_Data_Import_Batch__c record home.
+  @description Component used to create DataImport__c records on the related NPSP_Data_Import_Batch__c record home.
 -->
 
-<aura:component description="BGE_EntryForm">
+<aura:component>
 
+    <!--Public Attributes-->
     <aura:attribute name="labels" type="Map" description="Labels retrieved in Apex related to Schema information." access="public"/>
     <aura:attribute name="donorType" type="String" default="Contact1" description="Used to circumvent unhelpful labeling of Account1/Contact1" access="public"/>
     <aura:attribute name="recordId" type="Id" description="Populated by force:hasRecordId" access="public"/>
-    <aura:attribute name="dataImportFields" type="Object[]" description="" access="public"/>
+    <aura:attribute name="record" type="DataImport__c" description="Object" access="public"/>
+    <aura:attribute name="dataImportFields" type="Object[]" description="List of dynamic field objects to use in the field iteration" access="public"/>
+    <aura:attribute name="requiredFields" type="List" description="Easily accessible list of required fields" access="private"/>
```

Comment:
I'd probably drop "easily accessible" :)

Reply:
Good catch! I'd actually wanted to delete this altogether, so will fix that too 👍